### PR TITLE
fix: unable to use duration literals in inflights

### LINKED ISSUE #1629: Unable to use duration literals in inflights
### Community Note

> Please vote by adding a 👍 reaction to the issue to help us prioritize.
> If you are interested to work on this issue, please leave a comment.

### I tried this:
`wing test` this:
```js
bring cloud;

new cloud.Function(inflight () => {
  let sec = 10m.seconds;
  log("${sec} seconds");
}) as "duration";
```

### I expected this:

To print "600 seconds".

### Instead, this happened:

```
ReferenceError: $stdlib is not defined
```

### Component:

Compiler, SDK

### Wing Version:

0.5.37

### Node.js Version:

_No response_

### Platform(s):

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by eladcon**
Desugar duration literals and parse them as `duration.fromSeconds(val)` expressions.

Fixes #1629 

## Checklist

- [X] Title matches [Winglang's style guide](https://docs.winglang.io/contributors/pull_requests#how-are-pull-request-titles-formatted)
- [X] Description explains motivation and solution
- [X] Tests added (always)
- [ ] Docs updated (only required for features)
- [ ] Added `pr/e2e-full` label if this feature requires end-to-end testing

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

File: examples/tests/sdk_tests/std/duration.w [hunk 1]
```diff
@@ -2,10 +2,11 @@
 // seconds
 
 assert(12m.seconds == 12 * 60);
+assert(std.Duration.fromMinutes(12).seconds == 12 * 60);
```
Comment (staycoolcall911) [inline examples/tests/sdk_tests/std/duration.w:5]: The `std` namespace is implicitly imported in wing. Do we have to specify the namespace `std.Duration.fromMinutes` or does `Duration.fromMinutes` works as well?

Comment (staycoolcall911) [reply examples/tests/sdk_tests/std/duration.w:5]: I think we should get that to work without `std`. Let's see if someone from the team can come up with a direction how to achieve that

Comment (ekeren) [reply examples/tests/sdk_tests/std/duration.w:5]: yes the goal is to have this working without the `std`

Comment (ekeren) [reply examples/tests/sdk_tests/std/duration.w:5]: this is a Duration problem, not related to this 
opened 
https://github.com/winglang/wing/issues/2935

**Review by staycoolcall911 — APPROVED** [review]
Looks great, clean solution

**Pull request merged**